src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area, Drawable {
```

Comment:
```suggestion
public class Circle extends Figure {
```
давай імплементувати ці інтерфейси в класі Figure